api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -316,6 +335,8 @@ def test_github_delete_repository(
     )
     for feature in github_repository.project.features.all():
         assert FeatureExternalResource.objects.filter(feature=feature).exists()
+
+    print(feature_external_resource)
```

Comment:
I think you want to remove this print statement.

Reply:
Removed